backend/routes/document.js

```diff
@@ -0,0 +1,75 @@
+// npm Packages
+const express = require('express');;
+const bodyParser = require('body-parser');
+const router = express.Router();
+
+//aws
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const fileType = require('file-type');
+const bluebird = require('bluebird');
+const uuid = require('uuid');
+
+// Local imports
+const Document = require('mongoose').model('Document');
+
+AWS.config.update({
+    accessKeyId: "AKIAIRMIGEOQPEMEWWBQ",
```

Comment:
We'll definitely want to remove these. It's not an immediate danger since the repo is private... but yeah. We should see how we go about invalidating that key in the future.